source/_docs/pantheon-yml.md

```diff
@@ -47,6 +47,26 @@ The name of the nested directory is not configurable.
 
 For more information, see [Serving Sites from the Web Subdirectory](/docs/nested-docroot/).
 
+### Filemount Path
+Pantheon provides a [cloud-based filesystem](/docs/files/) to store user-generated content and other website files. By default, we create a symlink to this filesystem at `/sites/default/files` (Drupal) or `/wp-content/uploads` (WordPress), but you can change the location with the `filemount` variable:
+
+```
+filemount: /files
+```
+
+#### Considerations
+* Since relocating the filemount path involves multiple steps which must be done correctly and in unison, we suggest making this changes only in a [Custom Upstream](/docs/custom-upstream).
+* Specify the exact path; only a limited selection of paths are valid:
```

Comment:
can we combine this with first paragraph?

Reply:
I'm not sure I agree with this, since these are two separate points.